python/aswfdocker/data/versions.yaml

```diff
@@ -372,7 +372,7 @@ versions:
       ASWF_OCIO_CONFIGS_VERSION: "1.0_r2"
       ASWF_OCIO_VERSION: "2.2.1"
       ASWF_OIIO_VERSION: "2.4.9.0"
-      ASWF_OPENEXR_VERSION: "3.1.7"
+      ASWF_OPENEXR_VERSION: "3.1.8"
```

Comment:
Please jump straight to OpenEXR 3.1.9.

3.1.8 (and 3.1.7, I think) was found to have botched the .so library naming scheme and thus created a library incompatibility where it could not be correctly swapped for earlier 3.1.x. The release of 3.1.9 was meant to specifically fix this, and 3.1.7/3.1.8 should be considered toxic.

Reply:
Crap, never mind, I was thinking of imath. Apologies.